CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+# 0.7.0
+
+## Features
+
+* New option (`upstream_tag_template`) was added to the configuration file
+ to support more version schemes.
+* Packit is now able to be used from the distgit repository.
+  * You need to specify `upstream_project_url` to make it work.
+* New authentication scheme is possible in the configuration file.
+  * Support for multiple git services/instances.
+
+## Fixes
+
+* Verify create-archive action output.
+* Added support for `Source` in the spec-file.
+
+## Minor
+
+* Two new job-types (`copr_build_finished` and `copr_build_finished`) were added.
+* Rename `upstream_project_name` option to `upstream_package_name`.
```

Comment:
This is a deprecation change, make it first in the changelog please.

Reply:
What about moving to `Breaking changes` section?

Reply:
Tomas, you had written it's a "deprecation change" and then accepted it as a "breaking change"? It's not breaking anybody? (If one doesn't rename it in the config and use packit<0.7.0)

Reply:
OK, is the renaming of the section to `Deprecation changes` good for both of you?

Reply:
workforme